extract/news/yna_crawler.py

```
@@ -0,0 +1,125 @@
+        df = pd.DataFrame(news_output, columns=["post_time", "title", "content", "source", "link","keyword"])
+
+        # 데이터프레임을 parquet로 변환하여 메모리에서 처리
+        parquet_buffer = io.BytesIO()
+
+        # Pyspark에서 datetime이 깨지기 때문에 us 단위로 변환하여 저장
+        table = pa.Table.from_pandas(df=df)
+        pq.write_table(table, parquet_buffer, coerce_timestamps='us')
+        parquet_buffer.seek(0)
+
+        try:
+            s3.upload_fileobj(Fileobj=parquet_buffer, Bucket=BUCKET_NAME, Key=f_name)
```

Ruff (0.8.2)

`BUCKET_NAME` may be undefined, or defined from star imports

(F405)